cfbs/commands.py

```diff
@@ -587,7 +589,7 @@ def clean_command(non_interactive=False):
     modules = definition["build"]
 
     def _someone_needs_me(this) -> bool:
-        if this["added_by"] == "cfbs add":
+        if "added_by" in this and this["added_by"] == "cfbs add":
```

Comment:
```suggestion
        if "added_by" not in this or this["added_by"] == "cfbs add":
```

Reply:
I think the default here should be to say that "someone needs me" if added_by is missing.